### ISSUE TITLE
feat(branding): 🥒 Embrace the DiLL...pickle that is ;)

### ISSUE THREAD

**Issue by gsweats** [Ambratolm/dll-icons#1]
# Feature Request `(scope=branding)`

## Intro

- I was going to lie to you in the User Story below because it added humour to the sentence.
- Ultimately, I didn't want to start our relationship like that.
- So, I spent 40 more minutes writing a TLDR, and then a TLDR to the TLDR, and now reformatting this intro.
- In all seriousness, it's taken a full hour (`1h23m45s`).
- Some 🐥 became 🍗 in that time.
- ...and now 💩 in the time it's taken me to make the very difficult financial calculations below.
- My rate is typically **CAD $4,569.00 per hour**, very reasonable in today's Open Source marketplace.
  - `* 1.3958h ~= $6,377.41`
  - which, for you, my ultra cool good looking and popular friend, I'll lower to just **CAD $6,069.69**.
  - Pay or else..._(don't forget snitches get stitches)_... 👊☠️🚑 ... who am I kidding...
- it's yours here tonight for [🎙️ FREE NINETY NINE 🔊](https://www.youtube.com/watch?v=ylT16QB6Uig)!
- So, dim the lights, grab a drink, maybe some grass, and sit-yo-ass-down while I tell you the tail of **the Little Git Project that Could Have Awesome Branding**, who had **0 Issues**, until @GregSweats _(me! Your author this fine evening)_ showed up with his project polluting ways, greedily posting his nonsense to real projects in an effort to refine his Issue Creation/User Story Creation skillllllzzzzzzzzzz. 💤💤💤

## TLDR; for the TLDR; (HYPER SUMMARY)

> As an **Overly Verbose Stoner**
> ....a.k.a. `0$$-h4x0r-3l1t3-M$-hating-mofo-but-wait-they-own-GH-and-VSCode-and-OMG-I-am-a-M$-fanboy` ...
> I saw **dll**, which, naturally, my **brain auto-completed** to **"dill pickle emoji"** 🥒.
> I nearly-instinctively **filed a GitHub Issue** to suggest it.
> I saw **0 Issues** and thought "**how can that be? I must change this! I must make an impact! I might get a GitHub 'first issue created' badge (if there is such a thing)**".
> So I filed the ticket that has now taken an hour almost to write (**1-hour and 30-minutes**).
> and now **hope you will respond**, ye **"Good ol' Simple Works Farms 🥒🧑‍🌾**" ;).

## TLDR;
I was getting 420'd and rewrote the entire story down to this little tidbit:

> **As an Overly Verbose Stoner**, **I saw `dll`** and **thought of "dill"**, then thought "dill pickle".

_likely because of the two bags of Dill Pickle chips I bought this week which I look at each night as I prepare myself to compare the two, despite being different brands, and one being rippled while the other is smooth, like a pickle._

> ...then thought "`dill pickle emoji`"
> ...then thought it's worth a cheeky GitHub Issue
> ...and then thought you should **"Embrace the `DiLL`"**

_and now think I might have wasted like....40 minutes of my life at this point....I hope this hits your funny bone 🦴_

## Original, Overly Long User Story, Might not have even finished it...
_(although is the one above that much shorter?)_

- As a New-to-you "User?"
- who is a fan of emojis
- I have started using custom folder icons in windows for things like:
    - `captures` _(obs-recordings)_,
    - `screenshots`,
    - `pictures`,
    - `🚫 Screenshots`
      - _(a duplicate of `screenshots`, within a similar folder structure, but on a different drive...another reason I need icons...to help quickly mark which folders are what)_
    - etc.
- I've run out of even-close-to-meaning-something icons in the default Windows DLL.
    - _...and was heart-broken seeing Windows 11 didn't appear to change any icons in that DLL 💔_
    - _...and crushed there was nothing visually distinguishable enough to mean anything for more than 3-4 folders 😢_
- I finally took the dozen or so keystrokes it took to search for "windows folder dll" 🔎
- and then very quickly refining to "windows folder dll custom" 🕵️
    - ...to avoid all the dev docs and other DLL spammy nonesense.
- I saw an **oasis** in the desert of **SEO Optimized DLL garbage selling boner pills 💀**!
- That oasis, was you! It was always you, dear _(let me scroll up to see the package name again since I've been writing this Issue for like 20mins now when I should be getting 420'd instead)_
- 
- Dear `dll-icons` by `simple-works`, so modest I don't even know your Author's name since it's not in the URL and I have not looked at the Git Commits, just eagerly scrolled to the main part of the beautiful [README.md] 🤓 

## Y'all Come Back Now, Ya Here 👩‍🌾 

Sincerely yours,

@GregSweats

**Comment by gsweats** [comment]
# RESULTS Update#1 - We are Officially Issue #1!
_not sure about any badges yet...lol..._

> ![image](https://github.com/simple-works/dll-icons/assets/106463642/632a0ef8-82d2-4e5a-b57c-4980ba86608f)
